src/lang/yue.json

```
@@ -99,6 +99,6 @@
     "Add New Monitor": "新增監測器",
     "Quick Stats": "綜合數據",
     "markdownSupported": "可以用 Markdown",
-    "wayToGetFlashDutyKey": "您可以进入 协作空间 -> (选择一个 协作空间) -> 集成数据 -> 新增一个集成 页面，添加“自定义事件”获得一个推送地址，复制地址中的 Integration Key，更多信息前往{0}",
+    "wayToGetFlashDutyKey": "您可以进入 协作空间 -> (选择一个 协作空间) -> 集成数据 -> 新增一个集成 页面，添加“Uptime Kuma”获得一个推送地址，复制地址中的 Integration Key，更多信息前往{0}",
```

See https://github.com/louislam/uptime-kuma/blob/master/CONTRIBUTING.md#translations

```suggestion
    "wayToGetFlashDutyKey": "您可以进入 协作空间 -> (选择一个 协作空间) -> 集成数据 -> 新增一个集成 页面，添加“自定义事件”获得一个推送地址，复制地址中的 Integration Key，更多信息前往{0}",
```

---

done in this [commit](https://github.com/louislam/uptime-kuma/pull/3661/commits/2585053748529af2117e558450c722f73b3c34c4)